### PR TITLE
Add airtouch5 documentation

### PR DESCRIPTION
**Pull request by danzel**
## Proposed change

Add documentation for the airtouch5 integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/98136
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4594
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/airtouch5.markdown [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+### Cover
+
+Each zone has a cover entity to represent the state of the damper. The damper position can be controlled only if the zone is not temperature controlled.
+
+### Binary Sensor
+
+Each AC unit has binary sensors to show whether bypass or spill is active.
+
+Each zone has binary sensors to show if the battery is low or spill is active.
```
Comment (edenhaus) [inline source/_integrations/airtouch5.markdown]:
```suggestion
```

They will not be added with the linked PR. Please create a separate docs PR for them

File: source/_integrations/airtouch5.markdown [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ha_release: 2023.9
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@danzel'
+ha_domain: airtouch5
+ha_platforms:
+  - binary_sensor
+  - climate
+  - cover
```
Comment (edenhaus) [inline source/_integrations/airtouch5.markdown]:
```suggestion
  - climate
```

File: source/_integrations/airtouch5.markdown [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+title: AirTouch 5
+description: Instructions on how to integrate the AirTouch 5 A/C controller into Home Assistant.
+ha_category:
+  - Climate
+ha_release: 2023.9
```
Comment (edenhaus) [inline source/_integrations/airtouch5.markdown]:
```suggestion
ha_release: 2024.2
```

**Review by edenhaus — DISMISSED** [review]

File: source/_integrations/airtouch5.markdown [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+ha_config_flow: true
+ha_codeowners:
+  - '@danzel'
+ha_domain: airtouch5
+ha_platforms:
+  - climate
+ha_integration_type: integration
+---
+
+The AirTouch 5 integration allows you to control Ducted Air Conditioning Systems that are using the [AirTouch 5](https://www.airtouch.net.au/smart-air-conditioning/airtouch-5/) Controller.
```
Comment (c0ffeeca7) [inline source/_integrations/airtouch5.markdown]:
```suggestion
The AirTouch 5 integration allows you to control ducted air conditioning systems that are using the [AirTouch 5](https://www.airtouch.net.au/smart-air-conditioning/airtouch-5/) controller.
```
Tiny tweak. Applied sentence-style capitalization as this is not a proper noun, but a generic term. Based on Microsoft Style Guide on [capitalization](https://learn.microsoft.com/en-us/style-guide/capitalization).

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @danzel :+1:

**Pull request merged**